test/benchmark_harness_test.dart

```diff
@@ -24,7 +24,5 @@ class MockBenchmark extends BenchmarkBase {
   MockBenchmark() : super('mock benchmark');
 
   @override
-  void run() {
-    runCount++;
-  }
+  void run() => runCount++;
```

Comment:
Because this function is not just calling another function, and because the value of the expression is not actually used as the return value, I would leave this function with braces.